### PR TITLE
translate a term to Arabic

### PR DESCRIPTION
**Pull request by imanalhasani**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

-  Iman Al Hasani

## Language: 

-  Arabic

## Terms defined:

-  algorithmic bias
- 
-

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -650,7 +650,11 @@
     def: "አንድ ስልተ-ቀመር አንድን የተወሰነን ነገር ለማከናወን ሊከተሏቸው የሚገቡ እርምጃዎች ፣ ጥሰቶች ወይም ህጎች ስብስብነው
       \ አልጎሪዝም  በኮምፒተር ሳይንስ ውስጥ  የሂሳብ ችግሮችን ለመፍታት የተቀመጡ የኮምፒተር ፕሮግራም መመሪያዎች ስብስብ ነው
       \n"
-
+  ar:
+    term: "الخوارزمية المتحيزة"
+    def: >
+    .[الخوارزمية](#algorithm) النتائج المتكررة التي تظهر التحيز أو المعالجة المنحرفة الموجودة في
```
Comment (BatoolMM) [inline glossary.yml:656]:
```suggestion
     النتائج المتكررة التي تظهر التحيز أو المعالجة المنحرفة الموجودة في
    .[الخوارزمية(#algorithm)]
   
```

File: glossary.yml [hunk 2]
```diff
@@ -650,7 +650,13 @@
     def: "አንድ ስልተ-ቀመር አንድን የተወሰነን ነገር ለማከናወን ሊከተሏቸው የሚገቡ እርምጃዎች ፣ ጥሰቶች ወይም ህጎች ስብስብነው
       \ አልጎሪዝም  በኮምፒተር ሳይንስ ውስጥ  የሂሳብ ችግሮችን ለመፍታት የተቀመጡ የኮምፒተር ፕሮግራም መመሪያዎች ስብስብ ነው
       \n"
-
+  ar:
+    term: "الخوارزمية المتحيزة"
+    def: >
+     النتائج المتكررة التي تظهر التحيز أو المعالجة المنحرفة الموجودة في
+    .[الخوارزمية](#algorithm)
+
```
Comment (BatoolMM) [inline glossary.yml:658]:
```suggestion
```

File: glossary.yml [hunk 3]
```diff
@@ -650,7 +650,13 @@
...
       \ አልጎሪዝም  በኮምፒተር ሳይንስ ውስጥ  የሂሳብ ችግሮችን ለመፍታት የተቀመጡ የኮምፒተር ፕሮግራም መመሪያዎች ስብስብ ነው
       \n"
-
+  ar:
+    term: "الخوارزمية المتحيزة"
+    def: >
+     النتائج المتكررة التي تظهر التحيز أو المعالجة المنحرفة الموجودة في
+    .[الخوارزمية](#algorithm)
+   
+       .على سبيل المثال، قد تعطي خوارزميات وسائل التواصل الاجتماعي الأولوية أو الوسم للمحتوى الخاص بمجموعات معينة من المستخدمين
```
Comment (BatoolMM) [inline glossary.yml:659]:
```suggestion
     .على سبيل المثال، قد تعطي خوارزميات وسائل التواصل الاجتماعي الأولوية أو الوسم للمحتوى الخاص بمجموعات معينة من المستخدمين
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

File: glossary.yml [hunk 4]
```diff
@@ -650,7 +650,12 @@
     def: "አንድ ስልተ-ቀመር አንድን የተወሰነን ነገር ለማከናወን ሊከተሏቸው የሚገቡ እርምጃዎች ፣ ጥሰቶች ወይም ህጎች ስብስብነው
       \ አልጎሪዝም  በኮምፒተር ሳይንስ ውስጥ  የሂሳብ ችግሮችን ለመፍታት የተቀመጡ የኮምፒተር ፕሮግራም መመሪያዎች ስብስብ ነው
       \n"
-
+  ar:
+    term: "الخوارزمية المتحيزة"
+    def: >
+     النتائج المتكررة التي تظهر التحيز أو المعالجة المنحرفة الموجودة في
+    .[الخوارزمية](#algorithm)
+     .على سبيل المثال، قد تعطي خوارزميات وسائل التواصل الاجتماعي الأولوية أو الوسم للمحتوى الخاص بمجموعات معينة من المستخدمين
```
Comment (BatoolMM) [inline glossary.yml:658]:
```suggestion
      .على سبيل المثال، قد تعطي خوارزميات وسائل التواصل الاجتماعي الأولوية أو الوسم للمحتوى الخاص بمجموعات معينة من المستخدمين
```

File: glossary.yml [hunk 5]
```diff
@@ -650,7 +650,12 @@
     def: "አንድ ስልተ-ቀመር አንድን የተወሰነን ነገር ለማከናወን ሊከተሏቸው የሚገቡ እርምጃዎች ፣ ጥሰቶች ወይም ህጎች ስብስብነው
       \ አልጎሪዝም  በኮምፒተር ሳይንስ ውስጥ  የሂሳብ ችግሮችን ለመፍታት የተቀመጡ የኮምፒተር ፕሮግራም መመሪያዎች ስብስብ ነው
       \n"
-
+  ar:
+    term: "الخوارزمية المتحيزة"
+    def: >
+     النتائج المتكررة التي تظهر التحيز أو المعالجة المنحرفة الموجودة في
```
Comment (BatoolMM) [inline glossary.yml:656]:
```suggestion
      النتائج المتكررة التي تظهر التحيز أو المعالجة المنحرفة الموجودة في
```

File: glossary.yml [hunk 6]
```diff
@@ -650,7 +650,12 @@
...
+    .[الخوارزمية](#algorithm)
```
Comment (BatoolMM) [inline glossary.yml:657]:
```suggestion
      .[الخوارزمية](#algorithm)
```

**Comment by BatoolMM** [comment]
The spaces are fixed, thank you @imanalhasani!

**Review by BatoolMM — APPROVED** [review]

**Comment by elletjies** [comment]
@imanalhasani Thank you for contributing to Glosario in 2022! We are looking forward to receiving more PRs from you during 2023!

**Comment by elletjies** [comment]
@all-contributors please add @imanalhasani for translation

**Pull request merged**